orders/models.py

```diff
@@ -0,0 +1,22 @@
+from tkinter import CASCADE
+from django.db       import models
+
+from core.models     import TimeStampModel
+from products.models import Product
+from users.models    import User
+
+class Order(TimeStampModel):
+    user         = models.ForeignKey(User, on_delete=models.CASCADE)
+    order_status = models.CharField(max_length=100)
```

Comment:
```suggestion
    order_status = models.ForeignKey(max_length=100)
```
status도 테이블로 만들어서 데이터 관리해주세요.